test/decorator_test.ts

```diff
@@ -52,4 +52,15 @@ describe('decorators', () => {
     expectTranslate('@IMPLEMENTS(Z) class X implements Y {}')
         .to.equal(' @ IMPLEMENTS ( Z ) class X implements Y , Z { }');
   });
+
+});
+
+describe('annotation/decorator hack', () => {
+  it('should strip "Annotation" from type names', () => {
+    expectTranslate('@CONST class FooAnnotation {}').to.equal(' @ CONST const class Foo { }');
+  });
+  it('should strip "Annotation" from imports', () => {
```

Comment:
This reminds me that this feature can lead to some other subtle bugs when used the type is used for example,

```
class FooAnnotation extends BarAnnotation
```
but we can handle them as they arise.